### PR TITLE
billing info on dashboard

### PR DESCRIPTION
**Pull request by sulaiman-rtf**
image
Screenshot 2021-03-04 at 01 12 29

### CODE REVIEW

File: src/views/Dashboard.vue [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <template>
   <div>
+      <b-alert :show="billingInfo.trial_end_days < 20" dismissible style="margin: auto">You have {{ billingInfo.trial_end_days }} days left of your free trial. <b-link to="/billing/plans">Click here to purchase</b-link></b-alert>
```
Comment (stegru) [inline src/views/Dashboard.vue:3]: needs to be `billingoInfo && ...` - i'm getting null reference error when i first log in.

Also, `v-if` might be better than `:show`, so the things inside it get ignored completely.

File: src/views/Home.vue [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <template>
   <div>
     <b-alert variant="success" :show="displayMessage">
+        <h4>You have 6 days left of your free trial</h4>
```
Comment (stegru) [inline src/views/Home.vue:4]: I think you forgot to remove this

File: src/views/Dashboard.vue (hunk 1 above)
Comment (stegru) [reply src/views/Dashboard.vue:3]: Also, other checks is needed - if the days are `0` and `< 0` for "expires today" and "has expired"

File: src/views/Dashboard.vue [hunk 3]
```diff
@@ -130,6 +133,9 @@ export default {
                 }
             }
             return list;
+        },
+        trial_left: function () {
```
Comment (stegru) [inline src/views/Dashboard.vue:137]: `trial_left` isn't used

**Review by stegru — CHANGES_REQUESTED** [review]

File: src/views/Dashboard.vue (hunk 1 above)
Comment (sulaiman-rtf) [reply src/views/Dashboard.vue]: `v-if` is not applicable here because `:show` is `v-model` under the hood

**Review by stegru — APPROVED** [review]

**Pull request merged**